### PR TITLE
Bkushigian/fix gambit readme

### PR DESCRIPTION
**Pull request by BenTheKush**
# Update Gambit Documentation

Jira ticket: https://certora.atlassian.net/browse/CERT-3383?atlOrigin=eyJpIjoiMDQ0YmIwYTE1ZjU0NDE3ZGI0ZTBlNjA1NjM0NTBiZjAiLCJwIjoiaiJ9
[Link to generated documentation](https://certora-certora-prover-documentation--143.com.readthedocs.build/en/143/docs/gambit/gambit.html)

Note: this documentation is _now generated automatically from the Gambit README_. This will help keep docs in Gambit repo and this repo in sync.

### CODE REVIEW

**Review by chandrakananandi — APPROVED** [review]
Thank you so much!! This is FANTASTIC

**Review by urikirsh — APPROVED** [review]
Great, minor comments

File: docs/gambit/gambit.md [hunk 1]
```diff
@@ -401,37 +426,38 @@ This has the following structure:
...
-|:---------------------|:--------------------------------------------------------------------------------|
-| `--solc-base-path`   | passes a value to `solc`'s `--base-path` argument                               |
-| `--solc-allow-paths` | passes a value to `solc`'s `--allow-paths` argument                             |
-| `--solc-remappings`   | passes a value to directly to `solc`: this should be of the form `prefix=path`. |
+| Option                | Description                                                                     |
+| :-------------------- | :------------------------------------------------------------------------------ |
+| `--solc_base_path`    | passes a value to `solc`'s `--base-path` argument                               |
+| `--solc_include_path` | passes a value to `solc`'s `--include-path` argument                            |
+| `--solc_remappings`   | passes a value to directly to `solc`: this should be of the form `prefix=path`. |
+| `--solc_allow_paths`  | passes a value to `solc`'s `--allow-paths` argument                             |
```
[the remark this answers is not in the document]
Comment (BenTheKush) [reply docs/gambit/gambit.md]: Done

File: docs/gambit/gambit.md [hunk 2]
```diff
@@ -280,20 +305,20 @@ The configuration file is a JSON file containing the command line arguments for
...
         "@openzeppelin=node_modules/@openzeppelin"
     ],
 }
 ```
 
 In addition to specifying the command line arguments, you can list the specific
 mutants that you want to apply, the specific functions you wish to mutate, and
-more.  See the [`benchmark/config-jsons` directory](https://github.com/Certora/gambit/tree/master/benchmarks/config-jsons)
-for examples.
+more.  See the [`benchmark/config-jsons` directory][config-examples] for
```
[the remark this answers is not in the document]
Comment (BenTheKush) [reply docs/gambit/gambit.md]: Done

**Comment by BenTheKush** [comment]
@urikirsh I've updated with your suggestions (I committed directly because this doc is auto-generated from Gambit README)

**Review by mdgeorge4153 — APPROVED** [review]
Looks good to me!

**Pull request merged**